MBBSEmu/Configuration.cs

```diff
@@ -168,6 +181,25 @@ private int GetTimerHertz(string key)
             return timerHertz;
         }
 
+        private string GetLogLevel(string key)
+        {
+            var logLevel = ConfigurationRoot[key];
+            var logLevels = new[] {"Debug", "Info", "Warn", "Error", "Fatal"};
```

Comment:
We can probably use the nLog built in `LogLevel.FromString()` vs. defining them here.

https://nlog-project.org/documentation/v4.0.0/html/M_NLog_LogLevel_FromString.htm

Reply:
It's not very clear how tolerant it is --- i am using it from take the config variable and put it into NLOG but here was trying to make sure it was a valid log level as to not crash anything

Program.cs:
```
LogManager.Configuration.AddRule(LogLevel.FromString(configuration.FileLogLevel), LogLevel.Fatal, "fileLogger");
```

Reply:
Kept this as I think its the best way, I re-used it for both console and file